lib/child-writer/index.js

```diff
@@ -170,14 +182,52 @@ exports.write = (image, drive, options) => {
     }
   }
 
+  // Timeout function that only runs if we exceed the specified timeout,
+  // terminating the server.
+  let terminationTimeout = null
+
+  // If the child process is created successfully it will be stored here.
+  let maybeChild = null
+
+  /**
+   * @summary Reset the timeout countdown
+   * @function
+   * @private
+   *
+   * @param {ChildProcess} child - child process
+   *
+   * @example
+   * resetTerminationTimeout()
+   */
+  const resetTerminationTimeout = (child) => {
+    if (child) {
+      maybeChild = child
```

Comment:
I think you could avoid having `maybeChild` and simply invoke the functions below on `child`. If `child` was passed as an argument here, then it will remain accessible by the timeout handler by the time it gets called.

Reply:
Unfortunately `child` won't be passed in later calls to `resetTerminationTimeout` so it'll only work for the first five seconds assuming it doesn't get reset, so we need to store `child` for the later calls.